packages/jest-editor-support/src/Settings.js

```diff
@@ -55,7 +55,7 @@ export default class Settings extends EventEmitter {
 
     // Defaults for a Jest project
     this.settings = {
-      testMatch: ['**/__tests__/**/*.js?(x)', '**/?(*.)(spec|test).js?(x)'],
+      testMatch: ['**/__tests__/**/*.js?(x)', '**/?(*.)+(spec|test).js?(x)'],
       testRegex: '(/__tests__/.*|\\.(test|spec))\\.jsx?$',
```

Comment:
@orta we forbid to use both `testMatch` and `testRegex` while normalizing config and command line args – ideas why it is this way here?

Reply:
These are used to determine if we should show dots on a page etc, not passed forwards to Jest - so it won't conflict